### PR TITLE
Add node resources and logs

### LINKED ISSUE #219: Add node logs section
- Describe the feature
- Explain the log availability timeframe
- Etc

### LINKED ISSUE #227: Node resource display
Put the node resource display in KB/docs.

Track [core #1655](https://github.com/chainstack/core/issues/1655).

### PR DESCRIPTION
**Pull request by akegaviar**
Added node logs and resources sections.

Closes #227
Closes #219

### CODE REVIEW

File: docs/platform/view-node-resources-allocation.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+* Node — the storage allocated to your peer to maintain the state and a copy of the ledger.
+* Peer — the CPU and memory allocated to your peer.
+* Chaincode server — the CPU and memory allocated to the chaincode external builder and launcher. See also Hyperledger Fabric documentation: [External Builders and Launchers](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_launcher.html) and [Chaincode as an external service](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_service.html).
+
+Quorum:
+
+* Node — the storage allocated to your Quorum to maintain the ledger.
+* Geth — the CPU and memory allocated to Quorum Geth.
+* TM — the CPU and memory allocated to the Tessera transaction manager. See also Quorum documentation: [What is Tessera](https://docs.goquorum.com/en/latest/Privacy/Tessera/Tessera/).
```
Comment (easeev) [inline docs/platform/view-node-resources-allocation.md:37]:
```suggestion
* Transaction manager — the CPU and memory allocated to the Tessera transaction manager. See also Quorum documentation: [What is Tessera](https://docs.goquorum.com/en/latest/Privacy/Tessera/Tessera/).
```

Comment (akegaviar) [reply docs/platform/view-node-resources-allocation.md:37]: It's TM in the tooltip.

Comment (easeev) [reply docs/platform/view-node-resources-allocation.md:37]: @kutysam looks we need a small fix here https://github.com/chainstack/core/blob/master/server/core/api/serializers/prettymeta_field.py#L292

See https://github.com/chainstack/core/issues/1655#issuecomment-627143849

Comment (kutysam) [reply docs/platform/view-node-resources-allocation.md:37]: Thanks for alerting me on this. I have created a PR at https://github.com/chainstack/core/pull/1752 
Let's wait till tests pass.

File: docs/platform/view-node-logs.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+meta:
+  - name: description
+    content: View the logs of your nodes deployed with the Chainstack managed blockchain services.
```
Comment (jona-sc) [inline docs/platform/view-node-logs.md:4]: @akegaviar not sure we ever refer to it as '**the** Chainstack managed blockchain services'.

File: docs/platform/view-node-logs.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Click your project.
+1. Click your network.
+1. Click the node name.
+1. Click **Logs**.
+
+The timestamps in the log entries are displayed in your local time.
+
+The logs are stored for 7 days. Logs older than 7 days are automatically deleted.
+
```
Comment (jona-sc) [inline docs/platform/view-node-logs.md:23]: @akegaviar we should mention about the container selector. It's not finished yet but here's the story and design: https://github.com/chainstack/core/issues/1691

File: docs/platform/view-node-resources-allocation.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+Hyperledger Fabric:
+
+* Node — the storage allocated to your peer to maintain the state and a copy of the ledger.
+* Peer — the CPU and memory allocated to your peer.
+* Chaincode server — the CPU and memory allocated to the chaincode external builder and launcher. See also Hyperledger Fabric documentation: [External Builders and Launchers](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_launcher.html) and [Chaincode as an external service](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_service.html).
+
+Quorum:
+
+* Node — the storage allocated to your Quorum to maintain the ledger.
```
Comment (jona-sc) [inline docs/platform/view-node-resources-allocation.md:35]: 'Allocated to your Quorum'. Is it correct? Should it be 'Quorum node'?

File: docs/platform/view-node-logs.md (hunk 2 above)
Comment (akegaviar) [reply docs/platform/view-node-logs.md:4]: welp, I'm calling it **the** everywhere in other meta sections. Using **the** as a determiner here and other sections. I'll spend some time thinking on this over the weekend I guess and see if I should change this in all sections. Need to think.

I suggest we don't dwell on this in this patch.

File: docs/platform/view-node-logs.md (hunk 3 above)
Comment (akegaviar) [reply docs/platform/view-node-logs.md:23]: Done.

File: docs/platform/view-node-resources-allocation.md (hunk 4 above)
Comment (akegaviar) [reply docs/platform/view-node-resources-allocation.md:35]: Good catch. Fixed.

File: docs/platform/view-node-logs.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+meta:
+  - name: description
+    content: View the logs of your nodes deployed with the Chainstack managed blockchain services.
+  - name: keywords
+    content: blockchain node logs logging network
+---
+
+# View node logs
+
+You can view the logs for your [dedicated nodes](/glossary/dedicated-node) on the Growth, Business, and Enterprise <a href="https://chainstack.com/pricing/" target="_blank">subscription plans</a>.
```
Comment (easeev) [inline docs/platform/view-node-logs.md:11]: A bit too early to talk about Growth plan, we're going to release it in a few weeks only

File: docs/platform/view-node-logs.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Click your project.
+1. Click your network.
+1. Click the node name.
+1. Click **Logs**.
+
+For Hyperledger Fabric, you additionally filter the logs by:
+
+* Peer — the logs of your Hyperledger Fabric [peer](/blockchains/fabric#peers).
+* Chaincode server — logs of your chaincode external builder and launcher. See also Hyperledger Fabric documentation: [External Builders and Launchers](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_launcher.html) and [Chaincode as an external service](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_service.html).
```
Comment (easeev) [inline docs/platform/view-node-logs.md:23]: This is misleading as we don't use External Builders and Launchers. In our case Chaincode server is Docker running in Docker (Docker-in-Docker): https://hub.docker.com/_/docker/

File: docs/platform/view-node-logs.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+For Quorum, you additionally filter the logs by:
+
+* Geth — the logs of your instance of Quorum Geth.
+* Transaction manager — the logs of your instance of the Tessera transaction manager. See also Quorum documentation: [What is Tessera](https://docs.goquorum.com/en/latest/Privacy/Tessera/Tessera/).
+
+The timestamps in the log entries are displayed in your local time.
+
+The logs are stored for 7 days. Logs older than 7 days are automatically deleted.
```
Comment (easeev) [inline docs/platform/view-node-logs.md:32]: I'd use https://en.wikipedia.org/wiki/Log_rotation jargon here

File: docs/platform/view-node-resources-allocation.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+Corda:
+
+* Node — the CPU, memory, and storage allocated to your Corda node.
+* Database — the CPU, memory, and storage allocated to your [Corda node database](https://docs.corda.net/docs/corda-os/4.4/node-database.html).
+
+Hyperledger Fabric:
+
+* Node — the storage allocated to your peer to maintain the state and a copy of the ledger.
+* Peer — the CPU and memory allocated to your peer.
+* Chaincode server — the CPU and memory allocated to the chaincode external builder and launcher. See also Hyperledger Fabric documentation: [External Builders and Launchers](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_launcher.html) and [Chaincode as an external service](https://hyperledger-fabric.readthedocs.io/en/release-2.0/cc_service.html).
```
Comment (easeev) [inline docs/platform/view-node-resources-allocation.md:31]: See above

File: docs/platform/view-node-logs.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+* Chaincode server — logs of the service running your chaincode.
+
+For Quorum, you additionally filter the logs by:
+
+* Geth — the logs of your instance of Quorum Geth.
+* Transaction manager — the logs of your instance of the Tessera transaction manager. See also Quorum documentation: [What is Tessera](https://docs.goquorum.com/en/latest/Privacy/Tessera/Tessera/).
+
+The timestamps in the log entries are displayed in your local time.
+
+The logs are rotated daily and retained for 7 days before deletion.
```
Comment (easeev) [inline docs/platform/view-node-logs.md:32]: Ok, it was a bad idea to talk about rotation here as it's irrelevant.

```suggestion
The logs are retained for 7 days before deletion.
```

File: docs/platform/view-node-logs.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Click your project.
+1. Click your network.
+1. Click the node name.
+1. Click **Logs**.
+
+For Hyperledger Fabric, you additionally filter the logs by:
+
+* Peer — the logs of your Hyperledger Fabric [peer](/blockchains/fabric#peers).
+* Chaincode server — logs of the service running your chaincode.
```
Comment (easeev) [inline docs/platform/view-node-logs.md:23]:
```suggestion
* Chaincode server — the logs of the service running your chaincode.
```

**Pull request merged**